examples/guided.py

```diff
@@ -381,6 +377,16 @@ def perform_installation(mountpoint):
 						archinstall.log(' * Profile\'s post configuration requirements was not fulfilled.', fg='red')
 						exit(1)
 
+		# If the user provided custom commands to be run post-installation, execute them now.
+		if len(archinstall.arguments['custom-commands']):
+			for index, command in enumerate(archinstall.arguments['custom-commands']):
```

Comment:
Could we move this into a helper function, just to keep guided as small as possible?

Reply:
Which library would you like the function to be in? It's definitely doable but this is a very small amount of code to extract into a function.

Reply:
Done.